seeds/zeroex/polygon/zeroex_api_polygon_fills_sample.csv

```diff
@@ -0,0 +1,6 @@
+tx_hash,block_number,evt_index,contract_address,block_time,maker,taker,taker_token,taker_symbol,maker_token,maker_symbol,token_pair,taker_token_amount,taker_token_amount_raw,maker_token_amount,maker_token_amount_raw,type,affiliate_address,swap_flag,matcha_limit_order_flag,volume_usd,tx_from,tx_to
+0x7a5153e2730be0415e183068313ea846fa2c20079d52f511a77234e59e48bbd8,36840946,177,0xdb6f1920a889355780af7570773609bd8cb1f498,2022-12-15 0:54,0x00000012556e6973776170563300000000000000,0x1a7399d44bb438078eb8f0b69fac0b43de0a0865,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,WETH,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,USDC,USDC-WETH,0.003965,3.97E+15,5.181535,5.18E+06,NewBridgeFill,0x11ededebf63bef0ea2d2d071bdf88f71543ec6fb,TRUE,FALSE,5.184856364,0x1a7399d44bb438078eb8f0b69fac0b43de0a0865,0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31
+0xc90babc3ef5276672912386738c8241db6020e6496ea265844229d970ee5c644,36842188,379,0xdb6f1920a889355780af7570773609bd8cb1f498,2022-12-15 1:37,0x0000000a446f646f000000000000000000000000,0x3cb53363f84d221c9b3d60c79bd15d1756462f5b,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,USDC,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,USDT,USDC-USDT,64.246188,6.42E+07,64.242616,6.42E+07,NewBridgeFill,0x11ededebf63bef0ea2d2d071bdf88f71543ec6fb,TRUE,FALSE,64.29844283,0x3cb53363f84d221c9b3d60c79bd15d1756462f5b,0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31
+0x44145453d457fffc4ec1044dcb0b24cb7f34f864ca4886100644b85e5b529d38,36842188,326,0xdb6f1920a889355780af7570773609bd8cb1f498,2022-12-15 1:37,0x0000000a446f646f000000000000000000000000,0x3e18740f63c1d916a1d9760414887b6fce922337,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,USDC,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,USDT,USDC-USDT,65.395966,6.54E+07,65.392395,6.54E+07,NewBridgeFill,0x11ededebf63bef0ea2d2d071bdf88f71543ec6fb,TRUE,FALSE,65.44922099,0x3e18740f63c1d916a1d9760414887b6fce922337,0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31
+0x9e13cebdfe47029e6e0e6088a4a220a3cfa4a6647f79932b54c9c869438f7105,36842189,245,0xdb6f1920a889355780af7570773609bd8cb1f498,2022-12-15 1:37,0x0000000a446f646f000000000000000000000000,0xbef022479241534e6becec1c71d9b7b64ae2b0c8,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,USDC,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,USDT,USDC-USDT,65.420563,6.54E+07,65.416863,6.54E+07,NewBridgeFill,0x11ededebf63bef0ea2d2d071bdf88f71543ec6fb,TRUE,FALSE,65.47371025,0xbef022479241534e6becec1c71d9b7b64ae2b0c8,0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31
+0x1e65be3d7057c586043245d729b5f827ef06f10c1bab01ab880084f0abef79d7,36839746,315,0xdb6f1920a889355780af7570773609bd8cb1f498,2022-12-15 0:12,0x00000012556e6973776170563300000000000000,0xaf677145f8f03ecd25da06e290c59cb5a64aa163,0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6,WBTC,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,WETH,WBTC-WETH,0.0009,9.00E+04,0.01225203548,1.23E+16,NewBridgeFill,0x5bd68b4d6f90bcc9f3a9456791c0db5a43df676d,TRUE,FALSE,16.04134501,0xaf677145f8f03ecd25da06e290c59cb5a64aa163,0xdef1c0ded9bec7f1a1670819833240f027b25eff
```

Comment:
Moreover you may merge seeds in `polygon`, `optimism`, `ethereum` folder into a same seed file(if they have same columns), by adding a new column `blockchain`. This will save place(avoid repeat definition) in the `dbt_project.yml`

Reply:
we'd prefer to leave them in separate directories. let me know if that presents any problems